Tests/ViewInspectorTests/ViewModifiers/EnvironmentModifiersTests.swift

```diff
@@ -17,6 +17,15 @@ final class ViewEnvironmentTests: XCTestCase {
         let sut2 = EmptyView().environment(\.colorScheme, .light)
         XCTAssertEqual(try sut2.inspect().emptyView().environment(\.colorScheme), .light)
     }
+
+    func testClosureEnvironmentValue() {
```

Comment:
This test fails.
<img width="1404" alt="Screenshot 2023-06-14 at 11 14 41 AM" src="https://github.com/nalexn/ViewInspector/assets/1791049/c85e9013-7c85-4a40-aa6b-2247e779e832">